Jenkinsfile

```diff
@@ -7,11 +12,17 @@ def buildApps(appList) {
   }
 }
 
-def runningOn(machine) {
-  println "Stage running on:"
-  println machine
+def buildDocs(){
+  sh 'pip install git+ssh://git@github.com/xmos/xmosdoc'
```

Comment:
Maybe this should be versioned, xmosdoc still makes lots of major version upgrades with breaking changes.
It would be nice if this referred to a version (with semver parsing) rather than a specific tag. 
I'll create an issue in xmosdoc to set up a PyPI package so this can be installed as a versioned package.

Reply:
that makes sense, then I'll let it like this for now and and the pip version later.